### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by jarviscanada**

### CODE REVIEW

File: profile/profile.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+# 4) briefly talk about your qualifications (soft and tech) to justify why the hiring manager should hire your 
+summary: My name is Miguel Dario, and I Graduated from the Business Technology Management program at Ryerson University, where I was exposed to fields such as data analytics, project management, and networking. I have previously worked as an intern for Manawa Networks where I developed strong client management skills through providing IT solutions. In the same job, I was responsible for documenting various IT systems and standards. I have a passion for technology and enjoy learning about new technologies and how to make sense of it. I also enjoy interacting with a wide range of clients, learning about their business problems, and providing solutions to those through technology. I believe that these combined make me a great fit for a Business Systems Analyst role.
+
+skills:
+  # Please include 
+  #     1)soft skills (e.g. elicitation, communicaiton, collaberation, interpersonal, etc ) 
+  #     2) business tools (such as Excel, Viso, etc.)
+  Business Skills:
+    - Documentation
+    - Client Management
```
Comment (jarviscanada) [inline profile/profile.yaml]: Stakeholder Mgmt

File: profile/profile.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    - Presentation
+    - Requirements Elicitation
+    - Agile/Scrum
```
Comment (jarviscanada) [inline profile/profile.yaml:23]: add a couple more

File: profile/profile.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  # Please add more data related skills, such as Python, Tableau, R, Matlab, AI, ML, Statistics, etc.
+  # You can also add some programming skills, such as programming languages, GIT, Linux, Cloud Platform, tools, etc.
+  Technical Skills:
+    - Data Analytics
```
Comment (jarviscanada) [inline profile/profile.yaml]: Data Analytics with Python, Tableau, and R

Comment (jarviscanada) [reply profile/profile.yaml]: Visio

File: profile/profile.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    - R
+
+# List of Jarvis projects
+# Please describe your resume from a BSA perspective (e.g. requirement elicitation, scope, stakeholder, coordination, diagrams)
+jarvis_projects:
+  - name: "Linux Cluster Resource Monitoring App"
+    git_url: "https://github.com/jarviscanada/jarvis_data_eng_MiguelDario/tree/main/linux_sql"
+    # Focus on BSA work instead of development work
+    # Include the following keywords: BRD, SRD, Elicitation, stakeholder, reqruiement, analysis, RDBMS, SQL, data modeling, SLA
+    description: Designed and Implemented a resource monitoring system that records hardware specifications, and resource usage in real time, and store in a PostgreSQL RDBMS.
```
Comment (jarviscanada) [inline profile/profile.yaml]: more BSA keywords

File: profile/profile.yaml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  - name: "Python Data Analytics"
+    git_url: "/python_data_anlytics"
+    # Focus on BSA work instead of development work
+    # e.g. solve business problem, requirement, data sources, analytics, stakeholders, documentation
+    description: ""
+  - name: "Hadoop"
+    git_url: "/hadoop"
+    description: ""
```
Comment (jarviscanada) [inline profile/profile.yaml]: remove

File: profile/profile.yaml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+# List of professional experience
+# Please focus on transferable skills, such as soft skills and data related skills
+professional_experience:
+  #Write your Jarvis role as a professional experience rather than training.
+  - title: "Technology Alignment Intern"
+    company: "Manawa Networks"
+    duration: "Sep 2021 - Jan 2022"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    # Avoid the following terms: training, learning, etc. Instead, you should focus on the work, such as worked on XYY; Wrote ABC document; Analyzed QWE project;
+    description: ": Documented IT systems and standards for over 25 clients across various industries, Attended regular inter-department meetings to review current company standards, and suggest improvements in IT processes. Developed project management, and leadership skills through leading a project to migrate over 120 users to 2-factor authentication."
```
Comment (jarviscanada) [inline profile/profile.yaml]: Documented IT systems and standards for over 25 clients across various industries, Attended regular inter-department meetings to review current company standards, and suggested improvements in IT processes. Developed project management and leadership skills through leading a project to migrate over 120 users to 2-factor authentication."


stakeholder, requirements, more BSA keybords

File: profile/profile.yaml [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+  - title: "Tutor"
+    company: "XYZ Company"
+    duration: "2019"
+    description: "Donec mattis sed justo et sagittis. Vestibulum lacinia nulla ipsum. Curabitur imperdiet nibh vitae leo lacinia laoreet. Nullam accumsan, lectus ut maximus ultricies, augue justo egestas mi, vel bibendum felis."
```
Comment (jarviscanada) [inline profile/profile.yaml]: remove

File: profile/profile.yaml (hunk 1 above)
Comment (MiguelDario98) [reply profile/profile.yaml]: Done

File: profile/profile.yaml (hunk 2 above)
Comment (MiguelDario98) [reply profile/profile.yaml:23]: Done

File: profile/profile.yaml (hunk 3 above)
Comment (MiguelDario98) [reply profile/profile.yaml]: Done.

File: profile/profile.yaml (hunk 5 above)
Comment (MiguelDario98) [reply profile/profile.yaml]: Done.

File: profile/profile.yaml (hunk 7 above)
Comment (MiguelDario98) [reply profile/profile.yaml]: Done.

File: profile/profile.yaml (hunk 4 above)
Comment (MiguelDario98) [reply profile/profile.yaml]: Done.

**Pull request merged**